design-proposals/kubebmc.md

```diff
@@ -0,0 +1,156 @@
+# Overview
+
+This proposal introduces [KubeBMC](https://github.com/starbops/kubebmc), a BMC emulator for Kubernetes that facilitates the deployment of software/applications/platforms such as OpenShift and OpenStack - whose installers typically require communication with bare-metal out-of-band management protocols like IPMI and Redfish - in KubeVirt VMs for development, testing, and debugging purposes, similar to the functionality provided by [VirtualBMC](https://opendev.org/openstack/virtualbmc) and [sushy-emulator](https://opendev.org/openstack/sushy-tools) but within a Kubernetes context.
```

Comment:
There's a discussion about the CRD naming below but I wonder if the project itself needs to be something a little less generic to k8s and more specific to KubeVirt to avoid people getting the wrong idea here? `KubeVirtBMC`?

Reply:
A good idea, similar to the renaming of the CRDs, the project itself should also be identifiable as part of KubeVirt.

Reply:
`KubeVirtBMC` makes sense. Thanks for the suggestion.